spec/rails_helper.rb

```diff
@@ -0,0 +1,78 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
purpose of using this line?

Reply:
Bhai, by setting RAILS_ENV to 'test', it ensures that the tests run in the test environment rather than development or production.

Reply:
what will happen if user pass the RAILS_ENV while running the test?

Reply:
Bhai, i read somewhere, if RAILS_ENV is not set, it automatically sets it to test. But if a user passes a different environment (e.g., RAILS_ENV=development rspec), the value of RAILS_ENV will be overridden with the passed value, and tests will run in the specified environment.